OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/pdf/PDFDisplay.java

```diff
@@ -36,9 +36,13 @@ public JPanel drawPreviewOnJPanel() {
         Image generatedImage = PDFPreviewBuilder.buildPDFPreview(selectedPage, viewZoom);
         
         if (generatedImage != null) {
-            ImageIcon icon = new ImageIcon(generatedImage);
-            JLabel picLabel = new JLabel(icon);
-            pdfImagePanel.add(picLabel);
+            try {
+                ImageIcon icon = new ImageIcon(generatedImage);
+                JLabel picLabel = new JLabel(icon);
+                pdfImagePanel.add(picLabel);
+            } catch (OutOfMemoryError oome) {
```

Comment:
Isn't image just throwing an OutOfMemoryError and returning Null?  I'm working on figuring out how to increase the heap size in the executable currently, maven is new to me, the limit of 64m is fairly low with the size of uncompressed images and pdfs so increasing that should help out.